app/test/block_production_test.go

```diff
@@ -0,0 +1,57 @@
+package app
+
+import (
+	"testing"
+	"time"
+
+	appns "github.com/celestiaorg/celestia-app/pkg/namespace"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/stretchr/testify/suite"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+)
+
+func TestBlockProductionTestSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping block production test suite in short mode.")
+	}
```

Comment:
[Question]
As of https://github.com/celestiaorg/celestia-app/commit/9ea27d47e487debe3b44dcbf74c832a5940f47a8, I guess the test name also needs to be added to the Makefile so that it's not run in `race` mode. However, I am seeing the CI to be green. Does this mean that the test node no longer have races?

@rootulp is this expected?